dashboard/src/components/Tabs.vue

```diff
@@ -13,12 +12,15 @@ type Tab = {
   show: boolean;
 };
 
-const { tabs } = defineProps<{
+const { tabs, pname } = defineProps<{
   tabs: Tab[];
+  pname: string;
```

Comment:
It was hard for me to understand this refers to the parameter name. Not sure if `param` would be clearer, but I wouldn't mind being more specific in this case.